sources/US/UT/anybullelk-generalseason.json

```diff
@@ -1,15 +1,17 @@
 {
-   "name": "UT Bull Elk units",
```

Comment:
This data looks like it is versions yearly, so the thing to do is leave the existing file, and make a new one for 2017 with 2017 in the filename, and put year in the name.